lib/java-extras/src/main/java/org/triplea/io/FileUtils.java

```diff
@@ -180,13 +182,39 @@ public static URL toUrl(final Path file) {
       return Optional.empty();
     }
 
+    // try to read the file with default character encoding, if fails then fallback
+    // to UTF-8 then try ISO-8859_1
     try {
-      return Optional.of(Files.readString(fileToRead));
+      try {
+        return Optional.of(Files.readString(fileToRead));
+      } catch (final MalformedInputException e) {
+        log.info(
+            "Warning: "
+                + "unable to read file (character encoding problem), will next try UTF-8: {}, {}",
+            fileToRead.toFile().getAbsolutePath(),
+            e.getMessage());
+      }
+
+      try {
+        return Optional.of(Files.readString(fileToRead, Charsets.UTF_8));
+      } catch (final MalformedInputException e) {
+        log.info(
+            "Warning: "
+                + "unable to read file (character encoding problem), will next try ISO-8859_1: {}, {}",
+            fileToRead.toFile().getAbsolutePath(),
+            e.getMessage());
+      }
+
+      try {
+        return Optional.of(Files.readString(fileToRead, Charsets.ISO_8859_1));
```

Comment:
I know that Latin-1 is still commonly used in waaay too many places. But I don't think this is the right fallback handling to perform.
If for some arbitrary reason the encoding uses Latin-9 which is very similar except for 8 characters this will probably still work without any issues but the characters will simply be the wrong ones.

Also `Charsets` is a class from the google libraries, the default java way to do it is `StandardCharsets`.

Reply:
Good point on latin-8 vs latin-9.
- Better to have the content render than crash. 
- From what I've observed, any of the latin specific characters will not render well or at all correctly anyways. 
- I'd rather debug why a character is rendering weirdly than troubleshoot and train someone to use a UTF-8 text editor.